lib/core/src/Cardano/Wallet/Primitive/AddressDerivation/Byron.hs

```diff
@@ -405,6 +407,23 @@ instance PersistPrivateKey (ByronKey 'RootK) where
             _ ->
                 Left "Key input must be two hex strings separated by :"
 
+instance PersistPublicKey (ByronKey 'AccountK) where
+    serializeXPub (ByronKey k _ (Passphrase p)) =
+        hex (unXPub k) <> ":" <> hex p
+
+    unsafeDeserializeXPub kp =
+        either err (id . mkKey) (deserializeKey kp)
+      where
+        err _ = error "unsafeDeserializeXPub: unable to deserialize ByronKey"
+        mkKey (key, pwd) = ByronKey key (Index 0) pwd
```

Comment:
That's incorrect :thinking: , the derivation path for Byron addresses can be anything! It has to be serialized with the key itself.

Reply:
in the end, after using PersistKey it is no longer needed -> removed it